packages/api/orm/entity/TaskOrder.ts

```diff
@@ -0,0 +1,34 @@
+import { BaseEntity } from "./BaseEntity";
+import { Clin } from "./Clin";
+import { Column, Entity, ManyToOne, OneToMany } from "typeorm";
+import { FileScanStatus } from "../../models/FileMetadata";
+import { Portfolio } from "./Portfolio";
+
+@Entity()
+export class TaskOrder extends BaseEntity {
+  @Column({
+    type: String,
+    nullable: false,
+    length: 17,
+    comment: "TO numbers are 13 characters. TO modifications are 17 characters.",
+  })
+  taskOrderNumber: string;
+
```

Comment:
See Jeff's PR, there is a comment about the business rules for Task Order.
Link: https://github.com/dod-ccpo/atat-web-api/pull/467#discussion_r755612885

Reply:
Thanks for the note.